forms/gridfield/GridFieldDetailForm.php

```diff
@@ -752,6 +752,21 @@ public function getTemplate() {
 	}
 
 	/**
+	 * Get list of templates to use
+	 *
+	 * @return array
+	 */
+	public function getTemplates()
+	{
+		$templates = SSViewer::get_templates_by_class($this, '', __CLASS__);
```

Comment:
Right, `__CLASS__` refers to the base class where the method context was defined (http://php.net/manual/en/language.constants.predefined.php). Was wondering if this breaks on subclasses which wouldn't look for templates matching parent classes, but we're good.